build.gradle

```diff
@@ -6,13 +6,17 @@ buildscript {
 plugins {
     id 'net.neoforged.gradleutils' version '[2.0.18,)'
     id 'com.github.ben-manes.versions' version '0.42.0'
+    id 'net.neoforged.licenser' version '0.7.2' apply false
 }
 apply plugin: 'maven-publish'
 
+def rootProject = project
```

Comment:
Why is this being done, instead of referencing [`Project.getRootProject`](https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html#getRootProject--) (i.e. `rootProject`)?